gatsby-node.js

```diff
@@ -71,11 +100,26 @@ const createCareerPages = async ({ actions }) => {
       ? description.value || ''
       : '';
 
+    /**
+     * Ideally we create first the nodes from the API and then create the pages
```

Comment:
yeah, I think I'm going to refactor our personio api approach in that direction soon so we can have a single point where we generate our cards.